src/main/java/uk/gov/hmcts/reform/jps/domain/StatusHistory.java

```diff
@@ -37,7 +38,8 @@ public class StatusHistory {
 
     @JsonIgnore
     @ToString.Exclude
-    @ManyToOne(fetch = FetchType.EAGER, optional = false)
+    @LazyCollection(LazyCollectionOption.FALSE)
```

Comment:
I suppose this annotation is required only for collection

Reply:
Removed